lib/input/azure_queue_storage_config.go

```diff
@@ -64,15 +66,20 @@ Only one authentication method is required, ` + "`storage_connection_string`" +
 // AzureQueueStorageConfig contains configuration fields for the AzureQueueStorage
 // input type.
 type AzureQueueStorageConfig struct {
-	StorageAccount          string `json:"storage_account" yaml:"storage_account"`
-	StorageAccessKey        string `json:"storage_access_key" yaml:"storage_access_key"`
-	StorageSASToken         string `json:"storage_sas_token" yaml:"storage_sas_token"`
-	StorageConnectionString string `json:"storage_connection_string" yaml:"storage_connection_string"`
-	QueueName               string `json:"queue_name" yaml:"queue_name"`
+	StorageAccount           string `json:"storage_account" yaml:"storage_account"`
+	StorageAccessKey         string `json:"storage_access_key" yaml:"storage_access_key"`
+	StorageSASToken          string `json:"storage_sas_token" yaml:"storage_sas_token"`
+	StorageConnectionString  string `json:"storage_connection_string" yaml:"storage_connection_string"`
+	QueueName                string `json:"queue_name" yaml:"queue_name"`
+	DequeueVisibilityTimeout string `json:"dequeue_visibility_timeout" yaml:"dequeue_visibility_timeout"`
+	MaxInFlight              int32  `json:"max_inflight" yaml:"max_inflight"`
```

Comment:
Minor nit pick: this should be `max_in_flight` in order to match similar fields elsewhere.

Reply:
Ok, sorry, I've looked at the `nats`, it has it like this, I'll change it in 1 min

Reply:
It's fixed. Thanks, @Jeffail